### DISCUSSION TITLE
ObjectBrick Delete Event

### DISCUSSION THREAD

**Discussion by julescournut** [pimcore/pimcore#16154] [Q&A / Support] [answered]
Hello, I'm currently working on Pimcore version 11 and I want to add some code during ObjectBricks deletion (datahub configuration edit to remove all fields from objectBricks).

I can't seem to find any event during this deletion (admin/class/objectbrick-delete), how can I do this ? Is this even possible ?

Thanks for you help.

**Comment by kingjia90** [comment, marked as answer]
Hello, you are looking for these
https://github.com/pimcore/pimcore/blob/855cefd1a1ed67f8fa399eb042a5c9c7d8e72f2c/lib/Event/ObjectbrickDefinitionEvents.php#L49-L61

These are called by https://github.com/pimcore/admin-ui-classic-bundle/blob/d9aff7c5f8378b6bf9f30f1a4b672d48f4198e47/src/Controller/Admin/DataObject/ClassController.php#L1317 
which lead to https://github.com/pimcore/pimcore/blob/855cefd1a1ed67f8fa399eb042a5c9c7d8e72f2c/models/DataObject/Objectbrick/Definition.php#L493

**Reply by julescournut** [reply]
Oops, I don't have the latest version of Pimcore and these events were merged couple weeks ago. My bad, I should have checked the latest releases. Thank you !
